<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>470</width>
    <height>208</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lblCrypto">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>411</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Empty</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblText">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>47</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Text : </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="tbxText">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>251</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnConvert">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>30</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Convert</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblCryptoAES">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>411</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Empty</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>470</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
